match.md

```diff
@@ -241,3 +248,35 @@ If a `Match` includes `Lineup` objects on its `MatchTeam` objects and includes `
     <td>The result of the player's buzz, representing how many points the player's team received.</td>
   </tr>
 </tbody></table>
+
+## MatchQuestionBonus object
+
+<table class="fields"><tbody>
+  <tr class="required">
+    <th>question_number</th>
+    <td class="type">Number</td>
+    <td>Which bonus number this was. May or may not match the `question_number` of the `MatchQuestion` parent, depending on whether bonuses are paired with tossups or simply read in order (and if the latter, on whether any bonus-earning questions failed to lead to a bonus).</td>
+  </tr>
+  <tr class="required">
+    <th>parts</th>
+    <td class="type">Array&nbsp;`MatchQuestionBonusPart`</td>
+    <td>Information about each part of the bonus.</td>
+  </tr>
+</tbody></table>
+
+For bonuses that do not have distinguishable parts &mdash; e.g. "Name any <i>m</i> of the <i>n</i> things that&hellip;" or 30&ndash;20&ndash;10-style bonuses, treat the entire bonus as a single `MatchQuestionBonusPart`.
+
+## MatchQuestionBonusPart object
+
+<table class="fields"><tbody>
+  <tr class="required">
+    <th>controlled_points</th>
+    <th class="type">Number</td>
+    <td>Number of points earned by the controlling team. In all formats with which we are familiar, this must be a nonnegative integer.</td>
+  </tr>
+  <tr class="optional">
+    <th>bounceback_points</th>
+    <th class="type">Number</td>
+    <td>Number of points earned by the non-controlling team. May be omitted if bonuses do not bounce back, but should not be omitted merely because the non-controlling team earned no points. In all formats with which we are familiar, this must be a nonnegative integer, and can only be positive if `controlled_points` is `0`.</td>
```

Comment:
I tweaked this in 3d2d9c300288e90fefc8a880db9013543cfc538f. I do prefer to hedge with "in all formats with which we are familiar"; for instance, I can imagine some IESA-esque format adding a penalty for illegal conferring on a bonus.